functions/package.json

```diff
@@ -16,8 +16,8 @@
   },
   "dependencies": {
     "actions-on-google": "^2.0.x",
```

Comment:
The actions-on-google-nodejs version 2.1.x has been released. We should upgrade the version from 2.0.x to 2.1.x, if possible. Sorry for commenting against the unchanged code.

Reply:
Done